libdnf/rpm/package_query.cpp

```diff
@@ -2105,15 +2105,15 @@ void PackageQuery::filter_downgradable() {
 /// @param samename: Queue containing the block
 /// @param start_block: Start of the block
 /// @param stop_block: End of the block
-/// @param latest: Number of first packages in the block to add into the map.
-///                If negative, it's number of first packages in the block to exclude.
-static void add_latest_to_map(
+/// @param n_first: Number of first packages in the block to add into the map.
+///                 If negative, it's number of first packages in the block to exclude.
+static void add_first_to_map(
```

Comment:
What about `add_n_first_to_map`? Somehow only `latest` or `first` is not exactly what it dos.